pkgs/by-name/ae/aerospace/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  fetchzip,
+  gitUpdater,
+  lib,
+  stdenv,
+  versionCheckHook,
+}:
+
+let
+  ghUrl = "https://github.com/nikitabobko/aerospace";
+  # Version must be a literal string as package has a capital letter in it
+  # whilst nixpkgs forbids capitals in version names
+  ghReleaseVersion = "0.14.2-Beta";
+
+  pname = "aerospace";
+  appName = "AeroSpace.app";
+in
+stdenv.mkDerivation {
+  inherit pname;
+
+  version = lib.strings.toLower ghReleaseVersion;
+
+  src = fetchzip {
+    url = "${ghUrl}/releases/download/v${ghReleaseVersion}/AeroSpace-v${ghReleaseVersion}.zip";
+    hash = "sha256-v2D/IV9Va0zbGHEwSGt6jvDqQYqha290Lm6u+nZTS3A=";
+  };
+
+  installPhase = ''
+    mkdir -p $out/Applications
+    mv ${appName} $out/Applications
+    cp -R bin $out
+    mkdir -p $out/share
+    cp -R shell-completion $out/share
+    cp -R manpage $out/share/man
+  '';
```

Comment:
I'm not very familiar with Darwin stuff, but are you sure none of these steps are done automatically by `stdenv`?

Reply:
Some of them like bin yes, tho man pages and shell autocomplete no bc their name is not the one expected

I could modify the src so they are automatically detected

Reply:
Nah, it's alright like this.